### PR TITLE
Added mailhog support

### PR DESCRIPTION
**Pull request by cableman**

### CODE REVIEW

File: scripts/itkdev-docker-compose [hunk 1]
```diff
@@ -90,6 +93,11 @@ case "$cmd" in
     docker-compose port mariadb 3306 | cut -d: -f2
     ;;
 
+  open:mailhog)
```
[the remark this answers is not in the document]
Comment (cableman) [reply scripts/itkdev-docker-compose]: Added both

File: scripts/itkdev-docker-compose [hunk 2]
```diff
@@ -90,6 +93,11 @@ case "$cmd" in
...
+    url=http://0.0.0.0:$(docker-compose port mailhog 8025 | cut -d: -f2)
```
[the remark this answers is not in the document]
Comment (cableman) [reply scripts/itkdev-docker-compose]: Changed

File: scripts/itkdev-docker-compose [hunk 3]
```diff
@@ -32,6 +32,8 @@ Commands:
   sql:port:
                 Display the exposed MariaDB SQL server port.
 
+  open:url:
```
Comment (rimi-itk) [inline scripts/itkdev-docker-compose]: This should be `mailhog:url` and so on …

Comment (cableman) [reply scripts/itkdev-docker-compose]: Fixed

**Pull request merged**